pkg/auth/model/model.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/auth/model";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.auth.model;
+
+message UserData {
+    int64 ID = 1;
+    google.protobuf.Timestamp CreatedAt = 2;
+    string Username = 3;
+    // FriendlyName, if set, is a shorter name for the user than
```

Comment:
I don't think it's really necessary to document this parameter again.
I would add documentation on top of each message stating which go struct it models

Reply:
Done